README.md

```diff
@@ -45,6 +45,28 @@ an improvement to the code, for example).
 If you're having trouble getting your output to match the expected, post an
 issue.
 
+## Using srclib-ruby in Windows/Cygwin environment
+
+### Prerequisites
+Windows with Cygwin installed, I have tested it with 
```

Comment:
Remove use of the first person pronoun "I" from docs.

This line, for example, should read "Windows with Cygwin installed. `srclib-ruby` has been tested with:"